doc/specs/SCOOBE.md

```diff
@@ -0,0 +1,185 @@
+
+### 1.2. Key-Definitions/Concepts
+
+Here are definitions for words and acronyms found throughout this document to ensure clarity:
+
+- **OOBE:** Out of box experience – The users&#39; initial interactions with the product immediately after installing the product and/or launching the product for the first time.
+- **SCOOBE:** Second chance out of box experience - The users&#39; initial interactions with the product immediately after upgrading the product and/or launching the product for the first time after upgrading.
+
+### 1.3. Goals and Non-Goals
+
+Goals:
+
+- Create a guided prompt that exposes the user to a brief overview of the new features and/or improvements included with the latest version of PowerToys.
+
+Non-Goals:
+
+- Present a copy-and-paste replica of the repository release notes. This information needs to be readily consumable, often requiring visual demonstrations of the new behavior and/or functionality that&#39;s either not possible or not necessary to depict on the repository release notes.
+
+## 2. Definition of Success
+
+**2.1. Customers**
+
+The PowerToys SCOOBE is for existing and new power users and developers who are looking to tune and streamline their Windows experience for greater productivity and enhanced user experience. As the PowerToys customer base tends to be particularly biased against SCOOBE prompts in general, we need to present the PowerToys SCOOBE dialog in such a way that it provides immediate value to end-users to improve the likelihood of users discovering all the additions to PowerToys by making their way through the prompt.
+
+**2.2. Expected Impact: Customer, and Technology Outcomes**
+
+- **High Reliability** : Less than 0.1% crash rate.
+- **Increased Activation** : 50% or more adoption rate of new feature/utilities among PowerToys users who already utilize associated tools.
+- **High User Retention:** 25% or more active PowerToys users after 28 days of upgrade.
+
+## 3. Requirements
+
+The SCOOBE dialog builds off the currently implemented OOBE dialog originally drafted by [Niel&#39;s mock-up.](https://github.com/microsoft/PowerToys/issues/1285)
+
+**3.1. Functional Requirements**
+
+**3.1.1. Functional Requirements**  **Overview**
+
+| **No.** | **Requirement** | **Pri** |
+| --- | --- | --- |
+|1. | The SCOOBE dialog should launch immediately after PowerToys is updated. | P0 |
+|2. | The SCOOBE dialog should be contained inside the existing OOBE Dialog under its own &quot;What&#39;s New&quot; page of the dialog window. See figure 5.1.1. | P0 |
+|3. | The content for the SCOOBE dialog should be stored externally from the PowerToys application. **\*** | P0 |
+|4. | When the &quot;What&#39;s New&quot; page is opened, the content displayed should by loaded dynamically from the relevant information contained in the external storage described in 3.1.1.3 above. Assumes the user&#39;s device is connected to the internet. | P0 |
```

Comment:
This is also something that should be discussed because it makes the implementation and  testing overwhelming for our team that is too small to keep adding features that provide little benefit to the users but are a potential source of bugs and add complexity to the development and testing phases.
For the end user, the fact that the content is dynamic is irrelevant and actually a potential limitation.
What is the real motivation to want dynamic content that needs to be downloaded from the internet?
Why do we need the ability to update that content **after** a new version has been released?
If we open SCOOBE as soon as the new version is executed, the users will see it at that time and then dismiss it.
Changing that content afterward will result in not being seen by the users that have already checked SCOOBE.
Adding a notification mechanism to alert those users that SCOOBE has been updated is another of those things that the team can't afford to do.